namui-cli/src/services/wasm_watch_build_service.rs

```diff
@@ -0,0 +1,120 @@
+use super::{
+    bundle_metadata_service::BundleMetadataService,
+    runtime_project::{self, GenerateRuntimeProjectArgs},
+    rust_build_service::{BuildOption, BuildResult, RustBuildService},
+    rust_project_watch_service::RustProjectWatchService,
+    wasm_bundle_web_server::WasmBundleWebServer,
+};
+use crate::{cli::Target, debug_println, util::print_build_result};
+use std::{error::Error, path::PathBuf, sync::Arc};
+
+pub struct WasmWatchBuildService {}
+
+pub struct WatchAndBuildArgs {
+    pub project_root_path: PathBuf,
+    pub port: u16,
+}
+impl WasmWatchBuildService {
```

Comment:
I just thought all code is same among wasm_{windows/linux}_electron and wasm_unknown_web. Please confirm this.

Reply:
Good. I think it's okay.